### PR TITLE
Fixing issues found in #17

### PR DESCRIPTION
**Pull request by mone27**

### CODE REVIEW

**Comment by mone27** [comment]
@massimo1980 potresti per favore testare che funziona tutto

**Comment by lissyx** [comment]
@mone27 You might want to merge latest changes from  https://github.com/Common-Voice/commonvoice-fr/tree/master before fixing those

File: DeepSpeech/import_lingualibre.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -xe
 
 pushd $HOME/ds/
 	if [ ! -f "/mnt/sources/lingua_libre_Q385-ita-Italian_train.zip" ]; then
-		wget https://lingualibre.fr/datasets/Q385-ita-Italian.zip -O /mnt/source/lingua_libre_Q385-ita-Italian_train.zip
+		wget https://lingualibre.fr/datasets/Q385-ita-Italian.zip -O /mnt/sources/lingua_libre_Q385-ita-Italian_train.zip
```
Comment (lissyx) [inline DeepSpeech/import_lingualibre.sh]: I don't know why you need all that code, `bin/import_lingua_libre.py` already does perform download.

Comment (Mte90) [reply DeepSpeech/import_lingualibre.sh]: we didn't know and we added manually in the bash file.
We can remove it without any issue

**Comment by mone27** [comment]
removed lingualibre and tested everything. @Mte90 please merge

File: DeepSpeech/train_it.sh [hunk 2]
```diff
@@ -57,7 +57,6 @@ pushd $HOME/ds/
             --checkpoint_dir /mnt/checkpoints/ \
             --export_dir /mnt/models/ \
             --export_tflite \
-            --nouse_seq_length \
```
Comment (Mte90) [inline DeepSpeech/train_it.sh]: this parameter is important?

**Pull request merged**